packages/react-vapor/src/components/subNavigation/SubNavigation.tsx

```diff
@@ -33,39 +33,45 @@ export interface ISubNavigationProps
 
 const ISubNavigationPropsToOmit = keys<ISubNavigationProps>();
 
-export class SubNavigation extends React.PureComponent<ISubNavigationProps & React.HTMLAttributes<HTMLElement>> {
-    componentWillMount() {
-        this.props.onRender?.();
-    }
+export const SubNavigation: React.FunctionComponent<ISubNavigationProps & React.HTMLAttributes<HTMLElement>> = ({
+    onDestroy,
+    onRender,
+    selected,
+    defaultSelected,
+    onClickItem,
+    items,
+    className,
+    ...props
+}) => {
+    React.useEffect(() => {
+        onRender?.();
+        return () => onDestroy?.();
+    }, []);
+    const navProps = omit(props, ISubNavigationPropsToOmit);
 
-    componentWillUnmount() {
-        this.props.onDestroy?.();
-    }
-
-    render() {
-        const selected = this.props.selected || this.props.defaultSelected;
-        const navProps = omit(this.props, ISubNavigationPropsToOmit);
-        const items = map(this.props.items, ({id, link, label, disabled}: ISubNavigationItem) => (
-            <li key={id} className={classNames('sub-navigation-item', {'mod-selected': id === selected})}>
-                <a
-                    href={link || '#'}
-                    className={classNames('sub-navigation-item-link', {disabled})}
-                    onClick={(e: React.MouseEvent<HTMLAnchorElement>) => this.handleItemClick(e, id)}
-                >
-                    {label}
-                </a>
-            </li>
-        ));
+    const handleItemClick = (e: React.MouseEvent<HTMLAnchorElement>, id: string) => {
+        onClickItem?.(id);
+        return true;
+    };
 
-        return (
-            <nav {...navProps} className={classNames('sub-navigation', navProps.className)}>
-                <ul className="sub-navigation-items">{items}</ul>
-            </nav>
-        );
-    }
+    const navItems = map(items, ({id, link, label, disabled}: ISubNavigationItem) => (
+        <li
+            key={id}
+            className={classNames('sub-navigation-item', {'mod-selected': id === selected ?? defaultSelected})}
+        >
+            <a
+                href={link || '#'}

```

Comment:
`href={link ?? '#'}` maybe? not sure

Reply:
`href={link}` is preferable in order to not redirect if no link is defined